docs/sequence_diagrams/notification-actionFlow.plantuml

```diff
@@ -0,0 +1,108 @@
+@startuml
+' declate title
+title 10.5. Notification Action Flow
+autonumber
+
+' declare actors
+collections "Notification-Topic" as TOPIC_NOTIFY
+boundary "Central Event Processor" as CEP
+control "Action Agent" as AA
+database "CEP Store" as DB
+
+box "Central Services" #LightYellow
+participant TOPIC_NOTIFY
+end box
+box "Central Event Processing Service" #LightGreen
+participant CEP
+participant AA
+participant DB
+end box
+
+' start flow
+Group Action Flow
+    activate CEP
+    CEP -> AA: Recieves an action event
+    activate AA
+    AA -> AA
+    note right of AA #lightGrey
+        Evaluate action event
+    end note
+
+    alt **Action Object** exist
+        AA -> DB: Request **Event Object**  details
+        activate DB
+        deactivate DB
+        hnote over DB
+            event
+        end hnote
+        note right of AA #LightGrey
+            **Event Object** details;
+                Action details,
+                Hub notification details,
+                DFSP notification details.
+        end note
+        AA -> AA
+        note right of AA #LightGrey
+            Prepare notification details
+        end note
+        AA -> DB: Request **Action Object** by same **Event Object** details
+        activate DB
+        deactivate DB
+        hnote over DB
+            action
+        end hnote
+
+        alt No previous **Action Object** with same event detail
+            AA -> DB: Store **Action Object**
+            activate DB
+            hnote over DB
+                action
+            end hnote
+            AA <-- DB: Return result **actionId**
+            deactivate DB
+            AA -> AA
+            note right of AA #LightGrey
+                Schedule reset repetition for
+                this **actionId** as per rules
+            end note
+            ref over AA :  Notification Scheduler Flow {[[https://github.com/mojaloop/central-event-processor/tree/master/docs/sequence_diagrams/notification-schedulerFlow.svg Scheduler Flow]]} \n
+
+            TOPIC_NOTIFY <- AA: Update topic with notification
+            note left of AA
```

Comment:
Please apply the following:
1. Specify note color #yellow explicitly.
2. Move before TOPIC_NOTIFY <- AA
3. Whenever you use an object { }, make sure it is well-formed, e.g. { from: "SYSTEM } (no whitespace, no field label, no = inside JSON)

Reply:
Updated to the following
            note left of AA #yellow
                Notification topic:
                    {
                        from: "SYSTEM"
                    }
            end note 

- standards as per docs/SequenceDiagrams/seq-settlement-6.*
below sample from seq-setwindow-6.1.1
        note left of SSAPI #Yellow
            {
                "id": data.settlementWindowId,
                "state": data.settlementWindowStateId,
                "reason": data.reason,
                "createdDate": data.createdDate,
                "changedDate": data.changedDate
            }
        end note

Reply:
Please have a look at the following sections of the article:
https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/JSON#Arrays_as_JSON
https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/JSON#Other_notes